yb-voyager/src/query/queryissue/detectors_ddl.go

```diff
@@ -644,14 +644,16 @@ func (d *IndexIssueDetector) DetectIssues(obj queryparser.DDLObject) ([]QueryIss
 					))
 				} else if isHotspotType && idx == 0 {
 					//If first column is hotspot type then only report hotspot issue
-					hotspotIssues, err := reportHotspotsOnIndexes(param.ExprCastTypeName, obj.GetObjectType(), obj.GetObjectName())
+					//For expression case not adding any colName for now in the issue
```

Comment:
won't param.Colname be populated in this case?

Reply:
No, in case of expression param can be anything an expression of some casting on column or function call etc.. so in this case the colName is not determined and we don't have one.